app/src/main/res/values-pt-rBR

```diff
@@ -0,0 +1,401 @@
+    <string name="show_boot_screen">Mostrar tela de inicialização</string>
+    <string name="mac_address">Endereço MAC</string>
+    <string name="randomize_mac_address">Randomizar endereço MAC</string>
+    <string name="randomize_mac_address_summary">Quando ativado, um endereço MAC aleatório é usado a cada inicialização. Isso pode fazer com que alguns jogos pensem que estão sendo executados em um sistema diferente e podem interromper eventos baseados em tempo.</string>
+    <string name="generate_new_mac_address">Gerar novo</string>
+    <string name="generate_new_mac_address_description">Gerando um novo endereço MAC fará com que os jogos pensem que estão rodando em um sistema diferente.</string>
+    <string name="enable_jit">Ativar JIT</string>
+    <string name="enable_jit_summary">Melhora o desempenho, mas pode causar congelamentos ou travamentos.</string>
+    <string name="jit_not_supported">O JIT é suportado apenas em dispositivos de 64 bits.</string>
+    <string name="threaded_rendering">Renderização encadeada</string>
+    <string name="fps_counter_position">Posição do contador FPS</string>
+    <string name="threaded_rendering_summary">Melhora o desempenho em jogos 3D quando ativado, mas pode causar falhas gráficas.</string>
+    <string name="category_video">Vídeo</string>
+    <string name="category_video_summary">Filtro, renderização encadeada, contador de FPS</string>
+    <string name="filter">Filtro</string>
+    <string name="category_audio">Áudio</string>
+    <string name="category_audio_summary">Volume, latência, fonte do microfone</string>
+    <string name="enable_sound">Ativar som</string>
+    <string name="interpolation">Interpolação</string>
+    <string name="bitrate">Taxa de bits</string>
+    <string name="volume">Volume</string>
+    <string name="volume_percentage">%1$d%%</string>
+    <string name="audio_latency">Latência do Áudio</string>
+    <string name="microphone_source">Fonte do microfone</string>
+    <string name="controller_layout">Layout do Controle</string>
+    <string name="category_save_files">Salvar Arquivos</string>
+    <string name="category_save_files_summary">Salvar arquivo &amp; salvar locais do estado</string>
+    <string name="save_next_rom">Salvar arquivo próximo à ROM</string>
+    <string name="save_file_directory">Salvar diretório de arquivo</string>
+    <string name="save_state_location">Salvar local do estado</string>
+    <string name="input">Entrada</string>
+    <string name="input_summary">Mapeamento de teclas, layouts, feedback (resposta) tátil</string>
+    <string name="key_mapping">Mapeamento de teclas</string>
+    <string name="key_mapping_summary">Atribuir teclas às entradas de controle</string>
+    <string name="show_soft_input">Mostrar controle de toque</string>
+    <string name="vibrate_on_touch">Vibrar ao tocar</string>
+    <string name="vibration_strength">Força da vibração</string>
+    <string name="soft_input_opacity">Entrada de opacidade suave</string>
```

Comment:
This translation doesn't match the English translation. I suggest using "Opacidade do controlo virtual".

As you can see, I've used "controlo virtual" to translate "soft input". I don't know what name is usually used in portuguese for that, but I think this translation is clear enough and could be used whenever we want to reference "soft input".

Reply:
> As you can see, I've used "controlo virtual" to translate "soft input". I don't know what name is usually used in portuguese for that, but I think this translation is clear enough and could be used whenever we want to reference "soft input".

😅 How weird to translate "soft input" because it leads to "entrada suave", which would have to be "controle de toque".
Alright, I'll fix it for "Opacidade do controle virtual".